Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -46,6 +47,76 @@ public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAs
             return contactPoliciesSettings;
         }
 
+        public async Task<int?> GetUserIdAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            const string query =
+                @"select IdUser from [User] u where u.Email = @Email;";
+            var queryParams = new { Email = accountName };
+            return await connection.QueryFirstOrDefaultAsync<int?>(query, queryParams);
+        }
+
+        public async Task UpdateContactPoliciesSettingsAsync(Entities.ContactPoliciesSettings contactPoliciesSettings)
```

Comment:
You can use the following to know if the contact policies settings were updated
```suggestion
        public async Task<bool> UpdateContactPoliciesSettingsAsync(Entities.ContactPoliciesSettings contactPoliciesSettings)
```
If we use this suggestion we will need to save `ExecuteAsync` result:
```
var rowsAffected = await connection.ExecuteAsync(updateQuery, contactPoliciesSettings);
return rowsAffected > 0;
```

Reply:
that was my first aproach i think it was removed during some refactor, i dont know why, i will be back into that implementation again